guides/common/modules/proc_configuring-repositories-proxy.adoc

```diff
@@ -10,6 +10,24 @@ Select the option that corresponds with operating system and version you want to
 
 == [[repositories-rhel-8]]{RHEL} 8
 
+For configuring repositories in {EL} 8, you must enable `ruby:2.7` and `postgresql:12`.
+
+. Verify the module versions:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# dnf module list --enabled
+----
++
+If the version of modules is incompatible:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# dnf module reset ruby
+# dnf module reset postgresql
```

Comment:
For what it's worth we already have some guidance around resetting modules in `guides/common/modules/con_troubleshooting.adoc`, but AFAIK we don't use it in Satellite. The biggest challenge with postgresql is that it could mean there's an existing PostgreSQL 10 database and migrating that to 12 is more than just resetting the module.

Reply:
I have raised the new PR #1694 with all the changes you recommended above. :+1: 
There is a constant learning :smiley:

Reply:
Let me know if I need to close this PR.